### DISCUSSION TITLE
a bit confusing

### DISCUSSION THREAD

**Discussion by p3st-textures** [SquirrelJME/SquirrelJME#115] [RetroArch (RatufaCoat)] [answered]
2 more or less important questions

1. does this core now boot into to any flip-phone app or game at all on retroarch.
2. where should i put the squirreljme-summercoatrom.zip file is it even called right ? 
    do i have to do something else with it ?

would be very nice to know a bit more ...this is the only core in retroarch i can´t get running so far.
and its quite a long time since it was added a bit more information how to / or file structure would be wonderful ^^

anyways keep the work up

**Comment by XerTheSquirrel** [comment, marked as answer]
Hi!

To answer the following points:

1. Currently SquirrelJME is not functional in RetroArch, it should be sometime this year hopefully. I was not able to work on it much at all during the course of 2020.
2. The ROM will in the future be built-into the core so that there is no additional download or setup required, although replacing the built-in ROM would still be possible to do.

Still making good progress on this for this year, so hopefully that continues to go well! Thank you.

**Reply by p3st-textures** [reply]
hi there too 

1.Thanks for the answer , i was just wondering cuz the core is downloadable for quite a while now and thougt maybe..... i´m just to dumb to set it up ^^

2. Yeah , i looked into it and now know what the needed rom file is , i was thinking that i need the whole zip renamed or such thing 

anyways nice someone is working on something like this , cuz their where some very nice games on flip-phones only (Doom RPG as example) i like to play again , so thank you for that
